config/settings/common.py

```diff
@@ -58,6 +58,7 @@
     "feder.institutions",
     "feder.monitorings",
     "feder.cases",
+    "feder.cases_tags",
```

Comment:
I don't see a problem with such a large application granularity. Originally I wrote this in one application, but later it causes a problem in creating url eg. `reverse("cases: list")` (list of what? tags or case?). So I started to rewrite it as a sub-application. In the end, I wrote this as a separate app and I like it because CRUD around tags is contained together, CRUD around a case is contained together, and so on. You can easily navigate through the project in this way and `views.py` and `models.py` is not too long without in-consistent spliting.

We could rewrite all urls to make routing look like `<category>:<model>-<action>`, but now we have de facto `<model>: <action>`, which pushes us into separate applications for a separate model with our own life cycle.

I can see that for Django Rest Framework it would be mostly one ViewSets, but the current CBV based on the Django vanilla is adopted.

Reply:
Ok, it looks like it's enough of separate code to make dedicated app for it. Although making it as `cases` app part would also make sense, because logically they are tightly coupled, so that's where my question came from.